src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
Think about which variables should be local in the method and which should be class-level. Your `userService` variable will be created on each call of `login`. Can you avoid this?